### PR TITLE
Feature/181 code coverage

### LINKED ISSUE #181: Code Coverage
## User stories
As a developer I want to have a code coverage field in order to know how good my tests are.

## Acceptance Criteria
- [x] Code coverage should be displayed in readme
- [x] CI should calculate Code coverage

### PR DESCRIPTION
**Pull request by LucasDerReisende**
Fixes #181 

coverage might not be accurate, because did not run on other branches
Details about code coverage can be seen in coveralls

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Additional features are tested
- [x] Docs (code, wiki & diagrams) updated
- [ ] Breaking changes anounced
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [x] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

**Review by antonykamp — COMMENTED** [review]
I'm excited 🥳  Do you have a link for us to see the coveralls dashboard in order to detect missing lines 😇 If so, could you add it to the wiki and readme please 🙌 Thanks 😇

**Comment by LucasDerReisende** [comment]
I can add a wiki entry for coveralls, sure :)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+
+`dev` branch: ![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=dev)
+
+`main` branch: ![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=main)
+
+## Code Coverage
+
+`dev` branch: ![Code Coverage](https://img.shields.io/coverallsCoverage/github/BP2022-AP1/bp2022-ap1?branch=dev)
+
+`main` branch: ![Code Coverage](https://img.shields.io/coverallsCoverage/github/BP2022-AP1/bp2022-ap1?branch=main)
```
Comment (SaturnHafen) [inline README.md]: Please make the badges clickable by wrapping them into a link. That way one can directly open the correct coverage page

Comment (LucasDerReisende) [reply README.md]: Should be resolved now, using another solution

File: README.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # bp2022-ap1
 
+## CI & CD
+
+`dev` branch: ![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=dev)
+
+`main` branch: ![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=main)
```
Comment (SaturnHafen) [inline README.md]: The workflow can be wrapped into a link as well. Would you mind doing that? (At least link to the actions page)

Comment (LucasDerReisende) [reply README.md]: Is also added

File: README.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # bp2022-ap1
 
+## CI & CD
+
+`dev` branch: [![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=dev)](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml)
```
Comment (SaturnHafen) [inline README.md]:
```suggestion
`dev` branch: [![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=dev)](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml?query=branch%3Adev)
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+
+`main` branch: [![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=main)](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml)
```
Comment (SaturnHafen) [inline README.md]:
```suggestion
`main` branch: [![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=main)](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml?query=branch%3Amain)
```

**Review by SaturnHafen — APPROVED** [review]
Looks good!

**Review by antonykamp — APPROVED** [review]
Hypeeeee 🤝 🙌

**Pull request merged**